terraform/production/terraform-compliance/dynamodb.feature

```diff
@@ -0,0 +1,20 @@
+Feature: DynamoDB is used as our NoSQL database service
+  In order to improve security
+  As engineers
+  We'll use ensure our DynamoDB tables are configured correctly
+
+  Scenario: Ensure BackupPolicy tag is present
+    Given I have aws_dynamodb_table defined
+    Then it must contain tags
+    And it must contain BackupPolicy
+
+  Scenario: Ensure point in time recovery enabled
+    Given I have aws_dynamodb_table defined
+    Then it must contain point_in_time_recovery
+    And its enabled property must be true
+
+  Scenario: Ensure a maximum of 1 LSIs
```

Comment:
Can we have the same check for GSI?

Reply:
GSI is not being implemented in this API which is why I didn't add it. Let me know if I've understood something wrong